AnnoyancesFilter/Cookies/sections/cookies_specific.txt

```diff
@@ -211,11 +211,12 @@ freundin.de,utopia.de,mein-schoener-garten.de,eveningnews24.co.uk,rollingstone.d
 !
 ! START: iubenda CMP
 ! No action required ##
-crai-supermercati.it,centromedicoarcidiacono.it,usi.it,laboratoriodiegoangeli.it,analisiroma.it,edizioni.multiplayer.it,smartworld.it,pianteweb.com,cioccolatonippon.it,hwupgrade.it,iphoneitalia.com,mediaset.it,arduino.cc,virginradio.it##body #iubenda-cs-banner
+crai-supermercati.it,centromedicoarcidiacono.it,usi.it,laboratoriodiegoangeli.it,analisiroma.it,edizioni.multiplayer.it,pianteweb.com,cioccolatonippon.it,hwupgrade.it,iphoneitalia.com,mediaset.it,arduino.cc,virginradio.it##body #iubenda-cs-banner
 ||cdn.iubenda.com^$domain=crazygames.com
 ! No action required (scrolling is blocked) #$#
 synlab.it,dissapore.com,leganerd.com,siracusanews.it,tomshw.it,ansa.it,ilmessaggero.it,dilei.it,quifinanza.it,siviaggia.it,buonissimo.it,paginebianche.it,paginegialle.it,pgcasa.it,supereva.it,tuttocitta.it,virgilio.it#$##iubenda-cs-banner { display: none !important; }
 synlab.it,dissapore.com,leganerd.com,siracusanews.it,tomshw.it,ansa.it,ilmessaggero.it,dilei.it,quifinanza.it,siviaggia.it,buonissimo.it,paginebianche.it,paginegialle.it,pgcasa.it,supereva.it,tuttocitta.it,virgilio.it#$#html { overflow: auto !important; }
+||cdn.iubenda.com^$domain=smartworld.it
```

Comment:
Hi! Could be added to the existing rule:
```
||cdn.iubenda.com^$domain=crazygames.com|smartworld.it
```

Reply:
Hi :)
Uhm I think it's better to add it to `! No action required (scrolling is blocked) #$#` section because if you don't block `cdn.iubenda.com` in this case scrolling is blocked. On the contrary `crazygames.com` belongs to `! No action required ##` because if you don't block `cdn.iubenda.com` in this case scrolling isn't blocked.
Anyway if you prefer I can add it to `||cdn.iubenda.com^$domain=crazygames.com|`, let me know :)